### DISCUSSION TITLE
SEU modification dates issue ( right column )

### DISCUSSION THREAD

**Discussion by Makrinizakari** [codefori/vscode-ibmi#800] [IBM i]
Hi, When we are editing the code on VSC the modification dates that are stored/display in the SEU are all set to zero. Could this be a bug or something not implemented ? And if it is something not implemented, would it be possible to implement it?

![screenshoot](https://user-images.githubusercontent.com/56634895/183845064-d76919d6-5f62-4667-a7c0-2d28b5dcc0ae.png)

**Comment by worksofliam** [comment]
Check the configuration for a connection. You can enable source dates in there.

**Comment by Makrinizakari** [comment]
I tried to enable sources dates in there. But now i have an error for all my member in the source file. I am on the V6R1 version.
![2](https://user-images.githubusercontent.com/56634895/183910958-f387cd84-b6bb-4421-a11a-7577d5637f00.PNG)

**Reply by worksofliam** [reply]
@Makrinizakari I am afraid we don't support 6.1.

**Reply by Makrinizakari** [reply]
And would it be possible to support this version or not at all? Because unfortunately it is crucial for us to deliver on this version

**Reply by worksofliam** [reply]
@Makrinizakari sorry, we have no plans to support 6.1.
